en/docs/administer/changing-to-postgresql.md

```diff
@@ -1,97 +1,147 @@
 # Changing to PostgreSQL
 
-By default, WSO2 products use the embedded H2 database as the database
+By default, WSO2 Identity Server use the embedded H2 database as the database
 for storing user management and registry data. Given below are the steps
 you need to follow in order to use PostgreSQL for this purpose.
-
-!!! tip "Before you begin"
-    You need to set up PostgreSQL before following the steps to configure
-    your product with PostgreSQL. For more information, see [Setting up
-    PostgreSQL](../../administer/setting-up-postgresql).
     
 
 ### Setting up datasource configurations
 
 A datasource is used to establish the connection to a database. By
-default, `         WSO2_CARBON_DB        ` datasource is used to connect
-to the default  H2 database, which stores registry and user management
-data. After setting up the PostgreSQL database to replace the default H2
-database, either change the default configurations of the
-`          WSO2_CARBON_DB         `datasource, or configure a new datasource to point it to the new database as explained below.
+default, `WSO2_IDENTITY_DB` and `WSO2_SHARED_DB` datasources are used to connect
+to the default  H2 database. Here `WSO2_SHARED_DB` is the database which stores registry and user management
+data.`         WSO2_IDENTITY_DB        ` is a specific database for the identity server which stores
+identity related data. After setting up the PostgreSQL database. You can point the `WSO2_IDENTITY_DB` or 
+`WSO2_SHARED_DB` or both to that PostgreSQL database by following below instructions.
```

Comment:
`WSO2_IDENTITY_DB` and `WSO2_SHARED_DB` datasources are used to connect
to the default  H2 database. 
-   `WSO2_SHARED_DB`: This database stores registry and user management data.
-   `WSO2_IDENTITY_DB`: This is database stores identity-related data. 

After setting up the PostgreSQL database, you can point the `WSO2_IDENTITY_DB` or `WSO2_SHARED_DB` or both to that PostgreSQL database by following below instructions.

Reply:
done